tools/depends/native/swig/Makefile

```diff
@@ -9,6 +9,8 @@ LIBNAME=swig
 VERSION=4.0.2
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.gz
+SHA512=05e7da70ce6d9a733b96c0bcfa3c1b82765bd859f48c74759bbf4bb1467acb1809caa310cba5e2b3280cd704fca249eaa0624821dffae1d2a75097c7f55d14ed
+include ../../download-files.include
```

Comment:
the two include statements feel a bit weird. I know this was done via a script, but just raising that.

Reply:
ordering matters. The vars must be set before download-files.include is included.
it's either 2 includes or we have to move a combined line down here

Reply:
yeah, i guess as/if i pull all the version stuff into its own VERSION file can combine back after that.